api/include/opentelemetry/nostd/string_view.h

```diff
@@ -120,6 +120,20 @@ class string_view
     return substr(pos1, count1).compare(string_view(s, count2));
   };
 
+  size_type find(char ch, size_type pos = 0) const noexcept
```

Comment:
Looks good. Minor concern is - since we're adding it here, it would'be been ideal to add all overloads:
```cpp
constexpr size_type find(basic_string_view v, size_type pos = 0) const noexcept;
constexpr size_type find(CharT ch, size_type pos = 0) const noexcept;
constexpr size_type find(const CharT* s, size_type pos, size_type count) const;
constexpr size_type find(const CharT* s, size_type pos = 0) const;
```

Reply:
I'd like to avoid [Scope creep] (https://en.wikipedia.org/wiki/Scope_creep) and complaints related to complexity for those operations.